neptune/new/exceptions.py

```diff
@@ -885,3 +885,29 @@ def __init__(self, location: str):
 class ArtifactUploadingError(NeptuneException):
     def __init__(self, msg: str):
         super().__init__("Cannot upload artifact: {}".format(msg))
+
+
+class NeptuneUnsupportedArtifactFunctionalityException(NeptuneException):
+    def __init__(self, functionality_info: str):
+        message = """
+{h1}
+----NeptuneUnsupportedArtifactFunctionality-------------------------------------
+{end}
+It seems you are using Neptune Artifacts functionality "{functionality_info}" that is currently not supported.
```

Comment:
Please move "functionality_info" to new line so in result we have something like this:

```
It seems you are using Neptune Artifacts functionality that is currently not supported.

Wildcard characters (*,?) in location URI are not supported.
```